ImageGallery/src/org/sleuthkit/autopsy/imagegallery/ImageGalleryOptionsPanel.java

```diff
@@ -20,13 +20,11 @@
 
 import java.beans.PropertyChangeEvent;
 import java.util.EnumSet;
```

Comment:
copyright date

Reply:
Updated / corrected copyright, added serialization ID, and removed dead logger variable.